internal/claude/parser.go

```
@@ -674,3 +658,70 @@
 
 	return result
 }
+
+// parseFilesInParallel processes JSONL files in parallel for better performance
+func parseFilesInParallel(jsonlFiles []string, startTime time.Time, debug bool) []*models.SessionEvent {
+	numWorkers := runtime.GOMAXPROCS(0)
+	if numWorkers > len(jsonlFiles) {
+		numWorkers = len(jsonlFiles)
+	}
+	
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)